src/modules/thumbnail/thumbnail.attributes.ts

```diff
@@ -0,0 +1,13 @@
+export interface ThumbnailAttributes {
+  id: number;
+  fileId: number;
+  type: string;
+  size: number;
+  bucket_id: string;
```

Comment:
We should use Sequelize uppercase feature to make this camelCase automatically

Reply:
in the frontend it uses `bucket_id` and `bucket_file` in snake_case, I guess it was done this way because in drive-server it was returned this way. 
I imagine that in mobile it will be the same, I leave these two fields in snake_case to avoid unexpected errors?

Reply:
Leave them in both ways (snake, camel), but we should correct this on the frontend, the API should not depend on any platform implementation detail

Reply:
Yes, I agree. It's just not to break anything that is already done by returning the fields in a different format, as the changes are required on the front end. Later on we can add a task to make the modification on the different platforms.